_posts/2019-01-20-after-m1-eol.md

```diff
@@ -93,3 +93,6 @@ OpenMage is competing with solutions like Shopware, Sylius and Shopify.
 OpenMage is for Companies, which for diverse reasons would not switch to Magento 2 anyway, but would consider to switch to one of the other Platforms. For them, OpenMage is the more cost efficient solution, as it only leads to the price of an update, not a replatforming.
 
 
+---
+
+If you want to help us with maintenance, contact us on [Github](https://github.com/OpenMage/magento-lts), [Twitter](https://twitter.com/OpenMageProject) or via [Discord](https://discord.gg/EV8aNbU).
```

Comment:
if we want to have a specific footer on posts, then we should put it into the templates

Reply:
I have only added links to existing footer.

Feel free to update.

Reply:
this is factual wrong. In this file you edited there was no footer.

Reply:
- https://www.openmage.org/2022/11/09/release-19-4-19-and-20-0-17.html
- https://www.openmage.org/2022/05/19/release-19-4-16-and-20-0-14.html